.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing Guidelines
+
+Thanks for contributing to Dafny!  Github is the right place to discuss feature requests, report issues with Dafny, or contact the Dafny developers.
+
+Before reporting an issue here, consider whether it would be better handled in one of the following places:
+
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/dafny), an online Q&A website, for help with writing and proving Dafny programs.
+- [`dafny-lang/ide-vscode`](https://github.com/dafny-lang/ide-vscode), Dafny's VSCode plugin, for issues or feature requests specific to the plugin itself (however, issues with Dafny's LSP server should be reported here).
+- [`boogie-org/boogie-friends`](https://github.com/boogie-org/boogie-friends/), Dafny's Emacs mode, for Emacs-specific problems.
+- [`boogie-org/boogie`](https://github.com/boogie-org/boogie), Dafny's intermediate verification language, for questions and issues specific to Boogie.
+
+All other pull requests and issues can be submitted here.
+
+- For issues, please include your Dafny version number, any relevant code, and expected results.
+
+- For pull requests, consider updating <../RELEASE_NOTES.md> if your change is user-visible, and add tests if possible.
```

Comment:
```suggestion
- For pull requests, consider updating [`RELEASE_NOTES.md`](RELEASE_NOTES.md) if your change is user-visible, and add tests if possible.
```
This _might_ work.

Reply:
Isn't that the wrong path?

Reply:
I was hoping that is would be relative to the root, but I just tried it and you're right.